tests/functional/conftest.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import os
+from subprocess import check_call, check_output
+
+import pytest
+import yaml
+
+JUJU_CRED_DIR = "/home/ubuntu/.local/share/juju/"
+SNAP_CONFIG_DIR = "/var/snap/prometheus-juju-exporter/"
+TMP_DIR = "/tmp"
+SNAP_NAME = "prometheus-juju-exporter"
+
+
+def get_juju_data():
+    """Get juju account data and credentials."""
+    juju_controller_file = os.path.join(JUJU_CRED_DIR, "controllers.yaml")
+    juju_account_file = os.path.join(JUJU_CRED_DIR, "accounts.yaml")
+    assert os.path.isfile(juju_controller_file)
+    assert os.path.isfile(juju_account_file)
+
+    with open(juju_controller_file) as controller_file:
```

Comment:
Since parsing `accounts.yaml` requires the `current_controller` variable read from `controllers.yaml` file, I think it is better to keep them in one function.